cache/cache_test.cc

```diff
@@ -47,7 +51,7 @@ void dumbDeleter(const Slice& /*key*/, void* /*value*/) {}
 
 void eraseDeleter(const Slice& /*key*/, void* value) {
   Cache* cache = reinterpret_cast<Cache*>(value);
-  cache->Erase("foo");
+  cache->Erase(EncodeKey(123));
```

Comment:
A key is normally a string. Why is "foo" changed to 123?

Reply:
I agree. What I had in mind when I did this was that every type of cache would eventually require 16B keys. But actually this isn't true.

Reply:
I thought about this. The problem is that we'd like the test cases to be compatible with as many cache types as possible. Especially if they don't fundamentally depend on the length of the keys.

In the particular case of "foo", we could still work something out---we overload EncodeKey so it takes a string of length at most 16.